packages/python-packages/apiview-copilot/evals/tests/mention_summarize/open_parser_issue.yaml

```diff
@@ -0,0 +1,11 @@
+testcase: mention_summarize_open_parser_issue
```

Comment:
@marcarro please add a summarize scenario where an issue already exists. In this case, the summarize step should respond something like "Thank you for your feedback! It looks like there's already an existing issue for this: <link>".